main.py

```diff
@@ -0,0 +1,74 @@
+import tkinter as tk
+from tkinter import ttk
+from tkinter import filedialog
+import matplotlib.pyplot as plt
+import json
+
+# Set color values
+LIGHT_BLUE = '#e3f8fb'
+
+
+# Matplotlib logic
+class PlotView:
+    """A class that handles parsing the JSON file and visualizing the graph"""
+    def __init__(self, file_name):
+        """Getting the file path, opening it and calling a method to load the information"""
```

Comment:
NIce explanation but under the __init__ method we usually leave ''Constructor'' as the docstring. Feel free to leave it as it is